.github/workflows/zowe-explorer-ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       # Continue to run tests on the other systems if one fails
       fail-fast: false
       matrix:
-        node-version: [10.x, 12.x, 14.x]
+        node-version: [12.x, 14.x]
```

Comment:
Should we start including the NodeJS LTS version (16) here? 
Same for FTP, (and possibly the CICS extension😋 )

Reply:
we could add it in.  not sure how the tests will go, didn't we try it on next branch and saw issues?  If there are issues we may want to investigate them.

Reply:
Makes sense.
Maybe we leave it as is for now (to keep the builds passing) and open an issue for testing with node16 😋